Cargo.toml

```diff
@@ -29,11 +29,11 @@ rust-version = "1.85"
 
 [workspace.dependencies]
 anyhow = "1.0"
-arrow = "54.2.1"
-arrow-array = "54.2.1"
-arrow-cast = "54.2.1"
-arrow-json = "54.2.1"
-arrow-schema = "54.2.1"
+arrow = "55.0.0"
+arrow-array = "55.0.0"
+arrow-cast = "55.0.0"
+arrow-json = "55.0.0"
+arrow-schema = "55.0.0"
```

Comment:
I just updated to only hit the `arrow` crate: https://github.com/stac-utils/rustac/pull/702

Reply:
And most of the time you don't need all of the underlying crates: https://github.com/apache/arrow-rs/blob/cee5124a7c37d74ce84913a5410158657293eccd/arrow/Cargo.toml#L42-L54

Reply:
> If you use each underlying crate you minimize your dependencies and have faster compile times

Good point ... I'll revert (or maybe just keep this PR as a reversion). I was just apeing **duckdb-rs** but I'm coming to form an opinion that I shouldn't follow their patterns 🙃

Reply:
I think at some level it's personal preference, but I think for my _library_ crates, I want to keep my dependency tree as small as possible so that downstream libraries (like rustac!) that want to use `geoarrow` core functionality aren't forced into a single extra dependency they don't need.